identity/identity.go

```diff
@@ -59,7 +59,6 @@ func HandlerForHTTPClient(doAuth bool, cli HTTPClient, zebedeeURL string) func(h
 				log.DebugR(req, "authentication enabled, checking for expected tokens", logData)
 
 				if isUserReq || isServiceReq {
-
```

Comment:
this change will only make it worse when merging with `master` later (it causes the only conflict, I discovered)

please reinstate that line 👾

Reply:
lol who knew whitespace could cause that much damage.